infra/base-images/base-runner/install_java.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -u
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Install java in a platform-aware way.
+
+ARCHITECTURE=
+case $(uname -m) in
+    x86_64)
+      ARCHITECTURE=x64
+        ;;
+    aarch64)
+      ARCHITECTURE=aarch64
```

Comment:
We probably don't care too much for aarch64 for JAva either?

Reply:
Indeed we don't. I figure though it's better to make the images the same for now and we can worry about optimizing later.